cdmtaskservice/app_state.py

```diff
@@ -52,6 +52,8 @@ class AppState(NamedTuple):
     jobflow_manager: JobFlowManager
     kafka_checker: KafkaChecker
     allowed_paths: list[str]
+    condor_exe_path: Path
```

Comment:
having some comments here about what these App state vars are would be helpful

Reply:
Fixed